notes (#1240)
elasticio/elasticio.github.io: content/_releases/23/19.md

---
title: Product Update - v23.19
description: Details of the product update v23.19 from 10th of May 2023.
layout: rel
version: "23.19"
section: Product Updates
category: product-updates
releaseDate: 2023-05-10
redirect_from:
  - /releases/23.19.html
---

## Changes in accessibility to the `Recipe` feature
Starting from the next release `22.23` the `Recipe` feature will be available for the tenant-level of subscription. For other types of subscriptions, the `Recipe` feature will be available only if it was pre-paid in advance.

_Note: If you are interested to use the `Recipe` feature functionality please contact our support._

## New Features

### Ability to transform BigInteger values into a string

> * We added new `node_config` option `consume_raw_data`. This parameter configures the parsing behaviour in the webhook step. In case it is true, webhook step receives input message in raw structure.

Reviewer: to the Flow attributes

    * [API DOCS were updated](https://api.elastic.io/docs/v2#/flows/post_flows)
* We added ability to enable `raw` in the Send Sample request.
* We added `Consume Raw Data` configuration for step in the `Advanced Settings` menu.

> {% include img.html max-width="100%" url="/assets/img/RN/2319/BigInt.png" title="BigInt" %}

Reviewer: @andrewreshitko here

---
title: Product Update - v23.19
description: Details of the product update v23.19 from 10th of May 2023.
layout: rel
version: "23.19"
section: Product Updates
category: product-updates
releaseDate: 2023-05-10
redirect_from:
  - /releases/23.19.html
---

## Changes in accessibility to the `Recipe` feature
Starting from the next release `22.23` the `Recipe` feature will be available for the tenant-level of subscription. For other types of subscriptions, the `Recipe` feature will be available only if it was pre-paid in advance.

_Note: If you are interested to use the `Recipe` feature functionality please contact our support._

## New Features

### Ability to transform BigInteger values into a string

> * We added a new `node_config` option `consume_raw_data`. This parameter configures the parsing behaviour in the webhook step. In case it is true, the webhook step receives an input message in the raw structure.

Reviewer 2: ```suggestion
* We added a new `node_config` option `consume_raw_data` as a configuration option. This parameter configures the parsing behaviour in the webhook step. If selected, the webhook step receives an input message in the raw structure.
```
